.github/workflows/prepare-package-release.yml

```diff
@@ -12,18 +12,18 @@ jobs:
         runs-on: ubuntu-20.04
         steps:
             - uses: actions/checkout@v3
-
-            - name: Install PNPM
-              run: npm install -g pnpm@^6.24.2
+            - uses: ./.github/actions/cache-deps
+              with:
+                  workflow_name: prepare-package-release
+                  workflow_cache: ${{ secrets.WORKFLOW_CACHE }}
 
             - name: Setup PHP
               uses: shivammathur/setup-php@v2
               with:
                   php-version: '7.4'
 
-            - name: Install dependencies
-
-              run: pnpm install
+            - name: Install and Build
+              uses: ./.github/actions/install-build
```

Comment:
If it doesn't require build, You can skip this step but keep the caching step as `pnpm install` can still use it.

Reply:
Gotcha, thanks 😊 

Caching now properly added